extensions-core/azure-extensions/src/test/java/org/apache/druid/data/input/azure/AzureInputSourceTest.java

```diff
@@ -364,29 +370,34 @@ public void test_systemFields()
         (containerName, blobPath, storage) -> null
     );
 
-    Assert.assertEquals("azure://foo/bar", azureInputSource.getSystemFieldValue(entity, SystemField.URI));
-    Assert.assertEquals("foo", azureInputSource.getSystemFieldValue(entity, SystemField.BUCKET));
-    Assert.assertEquals("bar", azureInputSource.getSystemFieldValue(entity, SystemField.PATH));
+    assertEquals("azure://foo/bar", azureInputSource.getSystemFieldValue(entity, SystemField.URI));
+    assertEquals("foo", azureInputSource.getSystemFieldValue(entity, SystemField.BUCKET));
+    assertEquals("bar", azureInputSource.getSystemFieldValue(entity, SystemField.PATH));
   }
 
   @Test
   public void abidesEqualsContract()
   {
-    EqualsVerifier.forClass(AzureInputSource.class)
-                  .usingGetClass()
-                  .withPrefabValues(Logger.class, new Logger(AzureStorage.class), new Logger(AzureStorage.class))
-                  .withPrefabValues(BlobContainerClient.class, new BlobContainerClientBuilder().buildClient(), new BlobContainerClientBuilder().buildClient())
-                  .withPrefabValues(AzureStorage.class, new AzureStorage(null, null), new AzureStorage(null, null))
-                  .withNonnullFields("storage")
-                  .withNonnullFields("entityFactory")
-                  .withNonnullFields("azureCloudBlobIterableFactory")
-                  .withNonnullFields("inputDataConfig")
-                  .withNonnullFields("objectGlob")
-                  .withNonnullFields("scheme")
-                  .verify();
+    EqualsVerifier
+        .forClass(AzureInputSource.class)
+        .usingGetClass()
+        .withPrefabValues(Logger.class, new Logger(AzureStorage.class), new Logger(AzureStorage.class))
+        .withPrefabValues(
+            BlobContainerClient.class,
+            new BlobContainerClientBuilder().buildClient(),
+            new BlobContainerClientBuilder().buildClient()
+        )
+        .withPrefabValues(AzureStorage.class, new AzureStorage(null, null), new AzureStorage(null, null))
+        .withNonnullFields("storage")
+        .withNonnullFields("entityFactory")
+        .withNonnullFields("azureCloudBlobIterableFactory")
+        .withNonnullFields("inputDataConfig")
+        .withNonnullFields("objectGlob")
+        .withNonnullFields("scheme")
+        .verify();
```

Comment:
After double-checking, I believe this is just a personal preference to avoid large amounts of indentation white space. Is there any preference for the Druid project?

Reply:
Yes, since this is more of a personal preference, I think it can be reverted as it doesn't really affect readability. Best to keep the diff small.

In this case, there is no style preferred by Druid as such. But in general, I prefer to not include changes which do not improve upon the existing code in any way. Since all devs have their own personal styles, it can sometimes cause PRs to bloat up with unnecessary changes.

Hope that makes sense 🙂